package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "my-museum-tour",
-  "version": "0.1.0",
+  "version": "1.17.0",
```

Comment:
👍 
Have we really done 18 releases of MMT already?

Reply:
Small releases, but yes!